pkg/ddevapp/task.go

```diff
@@ -1,15 +1,27 @@
+// Copyright 2020 Drud Technology LLC
```

Comment:
There is no reason to add copyrights to every file, as there's a LICENSE file in the repository.

Reply:
It's suggested in that LICENSE file to add this comment to each file see at the end of the file. But I'm also fine not having it.

Reply:
We won't be adding it to every file. It also clutters up the PR.

Reply:
Okay, I'll remove it again as it's out of scope for this PR anyway.

Reply:
Will push the changes once the tests are passed.